lib/vsc/filesystem/posix.py

```diff
@@ -403,15 +403,24 @@ def populate_home_dir(self, user_id, group_id, home_dir, ssh_public_keys):
             '    . ~/.bashrc',
             'fi',
             ]
+        bashrc_text = [
+            '# do NOT remove the following lines:'
```

Comment:
oh, snap, this would've broken things quite nicely...

do we have any tests here?